src/wazuh_modules/syscollector/syscollector_bsd.c

```diff
@@ -278,15 +320,47 @@ char* sys_parse_pkg(const char * app_folder, const char * timestamp, int random_
                     free(parts);
                     free(_parts);
                 }
+                else if((fgets(read_buff, OS_MAXSTR - 1, fp) != NULL) && strstr(read_buff, "<string>")){
+                  char ** parts = OS_StrBreak('>', read_buff, 2);
+                  char ** _parts = OS_StrBreak('<', parts[1], 2);
+
+                  cJSON_AddStringToObject(package, "description", _parts[0]);
+
+                  for (i = 0; _parts[i]; i++) {
+                      free(_parts[i]);
+                      free(parts[i]);
+                  }
+                  free(parts);
+                  free(_parts);
+                }
             }
         }
+        if(void_file==0){
+          if (strstr(app_folder, "/Utilities") != NULL) {
+              cJSON_AddStringToObject(package, "source", "utilities");
+          } else {
+              cJSON_AddStringToObject(package, "source", "applications");
+          }
+          cJSON_AddStringToObject(package, "location", app_folder);
+        }
+        else{
```

Comment:
It is better to not include `version`, `group` and `description` when the file cannot be parsed because we cannot control what contain these fields.